pkg/objstore/azure/helpers.go

```diff
@@ -19,6 +19,11 @@ const DirDelim = "/"
 
 var errorCodeRegex = regexp.MustCompile(`X-Ms-Error-Code:\D*\[(\w+)\]`)
 
+func init() {
+	// This needs to be done at startup because the underlying variable is not thread safe.
```

Comment:
We should add another comment that explains why we do this. Maybe issue link.

Reply:
I saw that here: https://github.com/Azure/azure-pipeline-go/blob/dc95902f1d32034f8f743ccc6c3f2eb36b84da27/pipeline/core.go#L276-L283

Do you want me to add the link to the func comment?

Reply:
No, no. I meant why we are setting it to false and what leads us to make this decision.

Reply:
Or to the docs, of course: https://godoc.org/github.com/Azure/azure-pipeline-go/pipeline#SetForceLogEnabled